Dockerfile

```diff
@@ -4,8 +4,9 @@ RUN apt-get update && apt-get install python-tk python3-tk tk-dev -y
 
 WORKDIR /app
 
-COPY ./app/requirements.txt .
+# Copy all files and subdirectories from ./app to /app in the image
+COPY ./app /app
```

Comment:
This shouldn't be needed due to:
https://github.com/RedisVentures/ArXivChatGuru/blob/fea82247a681e78cd1be293080f4357abd50577f/docker-compose.yml#L8

Looks liek the aim here is to simply mount the application contents with a volume so you can edit/dev in realtime.

Reply:
Well, it is not the same a docker-compose for developing or testing purposes than a CI/CD workflow. If you want, we can do a Dockerfile for the ghcr build or we could have both, since if you map the volume it should use the files mapped.

Reply:
True -- good point keep this!